### DISCUSSION TITLE
About parameter "value" in analytics.logPurchase function

### DISCUSSION THREAD

**Discussion by QuangBinhDinh** [invertase/react-native-firebase#6875] [Q&A]
I'm using `logPurchase` function with 3 parameters for logging purchase: `currency`, `value` , `transaction_id`. I recognize there are other parameters including shipping cost, discount , etc . My question is which value of purchase should I pass, before other fees applied or after other fees applied. Thanks.

**Comment by twocs** [comment]
Analytics will let you log what you like. In general, you will need to decide what makes best sense for your business. Do you have other systems for tracking purchases? I'd suggest using similar accounting principles so that they are easily comparable.
